### DISCUSSION TITLE
statistics like tautulli or trakt year/alltime in review

### DISCUSSION THREAD

**Discussion by EdeAbreu23** [trakt/showly#833] [Ideas / Feature Requests]
hello,

i think it would be really nice to have an option to see the stats in a fun way with a lot of differents graphs that will show time spent this year, this month, this day and so on

Thanks!

**Comment by michaldrabik** [comment]
> hello,
> 
> i think it would be really nice to have an option to see the stats in a fun way with a lot of differents graphs that will show time spent this year, this month, this day and so on
> 
> Thanks!

If you go to Collection tab you can open some statistics from there (statistic icon in the right upper corner next to settings).

But yeah currently they are not too advanced just yet :)
